internal/deviations/deviations.go

```diff
@@ -224,4 +229,6 @@ var (
 		"Device skip isis restart-suppress check if value is true, Default value is false")
 
 	MacAddressMissing = flag.Bool("deviation_mac_address_missing", false, "Device does not support /system/mac-address/state.")
+
+	gribiIPv4EntryMACOverride = flag.Bool("deviation_gribi_ipv4_entry_mac_override", false, "Set to true for device not supporting programming a gribi flow with a next-hop entry of mac-address only, default is false")
```

Comment:
`gribiIMACOverrideWithStaticARP` I think is a better naming.

Reply:
Changed the deviation name.